packages/blocks/callout/src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import { BlockComponent } from "blockprotocol/react";
+import React, { CSSProperties, RefCallback, useCallback } from "react";
+
+import { EmojiIcon } from "./EmojiIcon";
+
+type AppProps = {
+  editableRef?: RefCallback<HTMLElement>;
+  icon?: string;
+  text?: string;
+};
+
+const wrapperStyle: CSSProperties = {
+  borderRadius: "0.25em",
+  width: "100%",
+  position: "relative",
+  padding: "0.5em",
+  background: "#f9fafc",
+  border: "1px solid #dee7f3",
+};
+
+const textStyle: CSSProperties = {
+  minHeight: "1.5em",
+  paddingLeft: "1.5em",
+};
+
+export const App: BlockComponent<AppProps> = ({
+  editableRef,
+  icon,
+  text,
+  entityId,
+  updateEntities,
+}) => {
+  const handleIconChange = useCallback(
```

Comment:
I removed this `useCallback`. I tend to wrap every callback into `useCallback` for consistency, even if this does not add much value in a given case. This practice leads to a slightly poorer performance in theory, but I haven't seen any benchmarks that would demonstrate anything significant. This blanket rule helps avoid inline fns in JSX for readibility and helps avoid bugs like `useEffect` going crazy . It also makes it supereasy to add `React.memo` anywhere at any point without refactoring things in a few places. So this _is_ a premature optimisation, but it's a continuous choice. Since we haven't discussed this in the past, happy to not add `useCallback` in the meantime (please correct me if I forget and do this again).